main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -481,6 +483,87 @@ object CompletionProvider {
     }.toList
   }
 
+  /**
+    * Format type params in the right form to be inserted as a snippet
```

Comment:
Could you add an example, e.g. Returns a string of the form `Name[Tparame1, ..]` (I think its like that?)